test/e2e/performanceprofile/functests/utils/utils.go

```diff
@@ -7,14 +7,14 @@ import (
 	"os/exec"
 	"time"
 
-	. "github.com/onsi/ginkgo"
-
 	testlog "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/log"
+
+	. "github.com/onsi/ginkgo/v2"
 )
 
 const defaultExecTimeout = 2 * time.Minute
 
-func BeforeAll(fn func()) {
+func CustomBeforeAll(fn func()) {
```

Comment:
Why did we change the name here ? to override an original BefroeAll ?

Reply:
This must be fix in next PR to avoid issues with that custom BeforeAll func